src/Donors/Migrations/addPhoneColumn.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Give\Donors\Migrations;
+
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class addPhoneColumn extends Migration
+{
+    /**
+     * @unreleased
+     *
+     * @throws DatabaseMigrationException
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $donorsTableName = "{$wpdb->prefix}give_donors";
+
+        try {
+            maybe_add_column(
+                $donorsTableName,
+                'phone',
+                "ALTER TABLE `$donorsTableName` ADD COLUMN `phone` varchar(50) NOT NULL DEFAULT '' AFTER `email`"
```

Comment:
This is nit-picky haha but can we change this to be after `name`?  I just like the idea of the required columns like email and name to be first and optional ones like phone to be later in the order. 

```suggestion
                "ALTER TABLE `$donorsTableName` ADD COLUMN `phone` varchar(50) NOT NULL DEFAULT '' AFTER `name`"
```

Reply:
Also, is there any particular reason you went with `varchar(50)` for the phone number?

Reply:
Resolved in fdfd6147a061b95246cd1ada9eca1e33ca1e5ac2

About the `varchar(50)`, I decide to use this value after checking the replies in this thread: https://stackoverflow.com/questions/723587/whats-the-longest-possible-worldwide-phone-number-i-should-consider-in-sql-varc

I believe this size should let this field future-proof in all possible scenarios discussed on this thread, but I'm open to suggestions.

Reply:
Sounds good, i'm cool with that just wanted to clarify how you arrived there 😄